### PR TITLE
show canvas on retina in a correct way

### PR DESCRIPTION
**Pull request by andreiglingeanu**
This is a split of #117. More details there.

### CODE REVIEW

File: dist/circle-progress.js [hunk 1]
```diff
@@ -157,6 +157,13 @@ License: MIT
             canvas.width = this.size;
             canvas.height = this.size;
             this.ctx = canvas.getContext('2d');
+
+            if (window.devicePixelRatio > 1) {
+                var scaleBy = window.devicePixelRatio;
+                canvas.style.width = canvas.style.height = [this.size, 'px'].join('');
```
Comment (kottenator) [inline dist/circle-progress.js]: I've read that current JS implementations do `123 + 'px'` as fast as `[123, 'px'].join('')`. Let's keep the code simple ;)

File: tests/tests.js [hunk 2]
```diff
@@ -177,6 +177,36 @@
...
+            * Mock devicePixelRatio
+            */
+            window.devicePixelRatio = 1;
+
+            var canvas = createCircle({
+                value: 0.75,
+                size: 50
+            });
+
+            assert.equal(50 + 'px', $(canvas).css('height'));
```
Comment (kottenator) [inline tests/tests.js]: use `$(canvas).height()` instead of `$(canvas).css('height')`

File: tests/tests.js [hunk 3]
```diff
@@ -177,6 +177,36 @@
...
+            * Mock devicePixelRatio
+            */
+            window.devicePixelRatio = 2;
+
+            var canvas = createCircle({
+                value: 0.75,
+                size: 50
+            });
+
+            assert.equal(50 + 'px', $(canvas).css('width'));
```
Comment (kottenator) [inline tests/tests.js]: Use `$(canvas).width()` instead of `$(canvas).css('width')`.

File: tests/tests.js [hunk 4]
```diff
@@ -177,6 +177,36 @@
...
+            * Mock devicePixelRatio
+            */
+            window.devicePixelRatio = 2;
+
+            var canvas = createCircle({
+                value: 0.75,
+                size: 50
+            });
+
+            assert.equal('50', $(canvas).width());
```
Comment (kottenator) [inline tests/tests.js]: Replace `'50'` with `50`

File: tests/tests.js [hunk 5]
```diff
@@ -177,6 +177,36 @@
...
+            * Mock devicePixelRatio
+            */
+            window.devicePixelRatio = 1;
+
+            var canvas = createCircle({
+                value: 0.75,
+                size: 50
+            });
+
+            assert.equal('50', $(canvas).height());
```
Comment (kottenator) [inline tests/tests.js]: Replace `'50'` with `50`

Comment (andreiglingeanu) [inline tests/tests.js]: I thought they need to have same type. I was wrong.

**Comment by kottenator** [comment]
Looks good! Let's wait untill tests passed and I'll merge it then.

**Comment by andreiglingeanu** [comment]
@kottenator We are good to go.

**Pull request merged**